### ISSUE TITLE
multiple_windows does not render in secondary window

### ISSUE THREAD

**Issue by aevyrie** [bevyengine/bevy#4378]
## Bevy version

main

## Operating system & version

Win11

## What you did

`cargo run --example multiple_windows`

## What you expected to happen

The gltf should render in both windows

## What actually happened

The model only shows in one window, but not consistently the same one

## Additional information

![image](https://user-images.githubusercontent.com/2632925/161112138-bcf688c7-d96a-4a96-9b64-0aff22be8370.png)
![image](https://user-images.githubusercontent.com/2632925/161116783-31a71db5-7b7c-4edc-b39b-3a94dff4533d.png)

**Comment by DJMcNab** [comment]
I can reproduce. Interestingly, it seems to arbitrarily select one window camera to work; I imagine that there's some hashmap somewhere causing this behaviour.

I'm assuming it's a camera issue, since the clear pass is running successfully.

**Comment by aevyrie** [comment]
Yeah, I just noticed the same thing and updated the ticket. 👍

**Comment by cart** [comment]
Yeah we changed from rendering all cameras of a given type to just the active one (which is why this used to work).
Historically we did these:
1. Prior to this pr: created a brand new camera type and manually updated the render graph. Each type (the default and the secondary camera) had its own active camera, which we selected.
2. Old version of this pr: rendered every camera with the same type (indiscriminately). Not the direction we should be headed in imo.
3. Merged version of this pr: render active cameras of a given type.

The missing piece here is choosing the "right" way to drive rendering. This is the "camera driven" vs "render target driven" discussion started in #4191.

This is next on my todo list, but I've been focused on merging other peoples' work for 0.7 for awhile now. I'm pretty close to being out of that hole though.

 I think we "need" active cameras with the current paradigm (and RenderTarget driven rendering). There are flavors of camera-driven where we don't (see #4191 for more info), but our current impl isn't that.
 
 If we can't merge a full solution before 0.7, I think the move is to re-add the "second window camera type / render graph" pattern to this example, but I _really_ want higher level render targets that "just work" asap.
